src/vs/workbench/contrib/issue/browser/baseIssueReporterService.ts

```diff
@@ -344,6 +475,65 @@ export class BaseIssueReporterService extends Disposable {
 			const { fileOnExtension, fileOnMarketplace } = this.issueReporterModel.getData();
 			this.searchIssues(title, fileOnExtension, fileOnMarketplace);
 		});
+
+		this.previewButton.onDidClick(async () => {
+			this.delayedSubmit.trigger(async () => {
+				this.createIssue();
+			});
+		});
+
+		this.addEventListener('disableExtensions', 'click', () => {
+			this.issueFormService.reloadWithExtensionsDisabled();
+		});
+
+		this.addEventListener('extensionBugsLink', 'click', (e: Event) => {
+			const url = (<HTMLElement>e.target).innerText;
+			windowOpenNoOpener(url);
+		});
+
+		this.addEventListener('disableExtensions', 'keydown', (e: Event) => {
+			e.stopPropagation();
+			if ((e as KeyboardEvent).key === 'Enter' || (e as KeyboardEvent).key === ' ') {
+				this.issueFormService.reloadWithExtensionsDisabled();
+			}
+		});
+
+		this.window.document.onkeydown = async (e: KeyboardEvent) => {
```

Comment:
follow up for later, these are things that are disposable (or can be make disposable) and should be registered to be disposed of.